{
  "uid" : "e6db447d354cb4e1",
  "name" : "Update a checklist with invalid position",
  "fullName" : "Trello API Checklists: Update a checklist with invalid position",
  "historyId" : "495d3449f2cbeef11eb83518139cd281",
  "time" : {
    "start" : 1653698556959,
    "stop" : 1653698557275,
    "duration" : 316
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: No error message was found\n",
  "statusTrace" : "  File \"/Users/runner/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Users/runner/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"tests/api/features/steps/common_steps.py\", line 47, in step_impl\n    assert error in context.response, 'No error message was found'\n",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 0,
  "retriesStatusChange" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: No error message was found\n",
    "statusTrace" : "  File \"/Users/runner/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Users/runner/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"tests/api/features/steps/common_steps.py\", line 47, in step_impl\n    assert error in context.response, 'No error message was found'\n",
    "steps" : [ {
      "name" : "Given I created a new card",
      "time" : {
        "start" : 1653698556960,
        "stop" : 1653698557123,
        "duration" : 163
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And I created a checklist on the card",
      "time" : {
        "start" : 1653698557123,
        "stop" : 1653698557231,
        "duration" : 108
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When I send a \"PUT\" request to \"/checklists/{checklist}\"",
      "time" : {
        "start" : 1653698557231,
        "stop" : 1653698557273,
        "duration" : 42
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "827bfbc1c2ef345",
        "name" : ".table",
        "source" : "827bfbc1c2ef345.csv",
        "type" : "text/csv",
        "size" : 36
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "Then the response contains the message \"invalid\"",
      "time" : {
        "start" : 1653698557274,
        "stop" : 1653698557274,
        "duration" : 0
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: No error message was found\n",
      "statusTrace" : "  File \"/Users/runner/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Users/runner/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"tests/api/features/steps/common_steps.py\", line 47, in step_impl\n    assert error in context.response, 'No error message was found'\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "And the status code is \"400\"",
      "time" : {
        "start" : 1653698557275,
        "stop" : 1653698557275,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 5,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "tag",
    "value" : "negative"
  }, {
    "name" : "feature",
    "value" : "Trello API Checklists"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ "negative" ]
  },
  "source" : "e6db447d354cb4e1.json",
  "parameterValues" : [ ]
}